### ISSUE TITLE
Stop updating AssemblyVersion

### ISSUE THREAD

**Issue by alextd** [FluffierThanThou/Pharmacist#11]
TL;DR you don't need to update AssemblyVersion in AssemblyInfo.cs, only AssemblyFileVersion

Since Smart Medicine references the Pharmacist dll, it's bound to the AssemblyVersion, so every update of AssemblyVersion means compatibility is broken.

This took a *while* to figure out why but I'm just listing what I found now:

"Two assemblies that differ by version number are considered by the runtime to be completely different assemblies" - https://docs.microsoft.com/en-us/dotnet/framework/app-domains/assembly-versioning

"If this number changes, other assemblies have to update their references to your assembly" "AssemblyFileVersion: Used for deployment. You can increase this number for every deployment" https://stackoverflow.com/questions/64602/what-are-differences-between-assemblyversion-assemblyfileversion-and-assemblyin

**Comment by FluffierThanThou** [comment]
thanks, I'll look into it!

**Comment by FluffierThanThou** [comment]
build script updated, AssemblyVersion should now be kept at `major.0.0`, where major only changes on RW version bumps or major changes to the mod.
